validation/static/menu.js

```diff
@@ -1,17 +1,21 @@
 "use strict";
 
-function showMenu() {
-    let menu = document.getElementById("options-menu");
-    if (menu.classList.contains("menu__none")) {
-        menu.classList.remove("menu__none");
-        menu.classList.add("menu__block");
-    } else {
-        menu.classList.add("menu__none");
-        menu.classList.remove("menu__block");
+document.addEventListener("DOMContentLoaded", () => {
+    for (let link of document.querySelectorAll('[data-cy="show-flex-button"]')) {
```

Comment:
So "flex" button refefers to 'flex as in "cirum" and not as in "-box". Because I thought this was a like a `button { display: flex; }` at first 😂 😂 😂